helpers/doesNotNeedPlugin.js

```diff
@@ -6,6 +6,11 @@ const doesNotNeedPlugin = ({ netlifyConfig, packageJson }) => {
   const { build } = netlifyConfig
   const { scripts = {} } = packageJson
 
+  if (!build.command) {
+    console.log("No build command specified in the site's Netlify config. Skipping plugin")
```

Comment:
should we maybe make this more noticeable somehow? i feel like it might get lost or like it might rly be worthy of a build failure. if we were going to fail the build tho, this check would have to come after/outside of `doesNotNeedPlugin`. therefore, we'd establish, ok, yes, so this site does need the plugin. it needs a build command for the plugin to work. _then_ we could do the check, and _then_ we should probably fail the build if it doesn't exist.

Reply:
The reason I didn't make it fail the build is because of the auto-install. It might be a site that somebody is building elsewhere, doing an export, then deploying. "`doesNotNeedPlugin`" wouldn't catch that scenario, because the command could be run elsewhere so wouldn;t show up in the config

Reply:
Maybe just making it bright red or something

Reply:
if doesNotNeedPlugin doesn't catch that scenario, wouldnt that just fail blindly anyways?

Reply:
It would fail because of this check, so if we moved it then it wouldn't fail.